src/Contracts/HasActivityLogDisplayName.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Threls\ThrelsActivityLog\Contracts;
+
+interface HasActivityLogDisplayName
```

Comment:
@ableta later on we may need other  methods to be added so the interface need to be general

```suggestion
interface ActivityLogContract
```